mempool/clist_mempool.go

```diff
@@ -80,6 +79,14 @@ var _ Mempool = &CListMempool{}
 // CListMempoolOption sets an optional parameter on the mempool.
 type CListMempoolOption func(*CListMempool)
 
+// Lane corresponds to a transaction class as defined by the application.
+// A lane is identified by a string name and has priority level.
+// Different lanes can have the same priority.
+type Lane struct {
```

Comment:
Does `Lane` need to be public?

Reply:
No. A lane is an implementation detail of CListMempool and it's not used outside of it.